### ISSUE TITLE
Opening a document from a mapped drive doesn't work.

### ISSUE THREAD

**Issue by bjakeway** [notepad-plus-plus/notepad-plus-plus#12030]
When I drag a document in a mapped drive into Notepad++, nothing happens.  When I navigate to the file itself without going through the mapped drive, the same file opens fine.  In the open file dialog, mapped drives don't show up.

### Steps to Reproduce the Issue
1. Drag a file found in a mapped drive into Notepad++.

### Expected Behavior
I would have expected the file to be opened in Notepad++

### Actual Behavior
Nothing.  The file didn't open.

### Debug Information
Notepad++ v8.4.4   (64-bit)
Build time : Jul 15 2022 - 17:54:42
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "P:\Clients\BC OGC\Upgrade\7.4.2\7.4.2 deprecations.txt"
Admin mode : ON
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 21H2
OS Build : 19044.1889
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.8)
    NppConverter (4.4)
    NppExport (0.4)

**Comment by rdipardo** [comment]
Duplicate of #12029 and #3847

Try letting `notepad++.exe` through the firewall: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3847#issuecomment-419162654

**Comment by guillcote** [comment]
I don't think this is related to the firewall.  I solve this issue on my computer by deinstalling and reinstalling from a direct download from the notepad++ site.

**Comment by guillcote** [comment]
@bjakeway, did you install with choco or from a shell or a command line windows?

**Comment by bjakeway** [comment]
I installed this a while ago.  The problem seemed to start today after upgrading Notepad++.

Sincerely,

Bruce Jakeway

**Comment by guillcote** [comment]
> I installed this a while ago. The problem seemed to start today after upgrading Notepad++
How did you upgrade?
Was it with choco?
Directly in Notepad++?
Otherwise?

Did you try to uninstall and te with run as administrator in file explorer?

**Comment by bjakeway** [comment]
I upgraded directly with Notepad++.

There was a Windows update this morning as well which may have caused it.  However, both TextPad and NotePad work with mapped drives, so I don’t think it’s the Windows update that caused it.

Sincerely,

Bruce Jakeway
+1-587-525-5391

**Comment by guillcote** [comment]
This is the comment that make me think the update could have cause it: 

https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3847#issuecomment-364809636

I had the same issue when I installed with choco.  When I uninstalled and installed again running the binary in file explorer, the issue was gone, I expect it should also solve your issue.

**Comment by bjakeway** [comment]
Uninstalling NotePad++ and reinstalling it worked fine.  And I’m happy that NotePad++ allows you to keep customizations even when you’re uninstalling it. 😊

Thanks for your help!

Sincerely,

Bruce Jakeway
+1-587-525-5391

**Comment by guillcote** [comment]
> Uninstalling NotePad++ and reinstalling it worked fine. And I’m happy that NotePad++ allows you to keep customizations even when you’re uninstalling it. 😊 Thanks for your help! Sincerely, 

I am happy to read that your issue is fixed.  You are welcome.

However, I think there is still a problem with the updater, someone need to do the update you did and see if he has the same problem you have to  confirm if we have a problem with the updater,  We should keep this defect open until we have the result of that test.
